5.2cup_of_join.py

```diff
@@ -0,0 +1,11 @@
+def cup_of_join(*lists, sep='-'):
+    """
+    get unknown lists numbers and join them by the sep
+    :param lists: the lists to join
+    :param sep: the seperator to join by default is '-'
+    :return: the joined lists
+    """
+    if not lists:
```

Comment:
1. No documentation for None case in function doc
2. check more explicitly than 'not'